magento/adobe-stock-integration
# #553 - Hide ImageName Field

Do not show image name field in license confirmation popup if image preview was saved before.

Now the user sees a confirmation popup without the ability to specify image name. 
I also stored the 'this.preview().buyCreditsUrl' in a variable in the right context as it changes in the 'confirm' function what returns 'TypeError: this.preview is not a function' when you click on 'Buy Credits' button.





### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#<issue_number>: Issue title
2. ...

### Manual testing scenarios (*)

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Click on an image to expand the preview
5. Click on "Save Preview" button
6. Reopen images grid
7. Click on "License" button for the same image
You will see a confirmation popup without the ability to specify image name. If image hasn't been saved before then you will see the same popup but will be present and the input to specify the image name.
